### ISSUE TITLE
About the setting of batchnorm2d

### ISSUE THREAD

**Issue by xingshuojing** [yuchendoudou/TODE#5]
Dear Dr. Chen, when I reproduced the code, I found an issue with the batchnorm2d module in the "tode" file. In the source code, it is written as batchnorm2d(64, 64), which sets the value of eps to 64. However, theoretically, eps should be a very small value. So, after correcting this problem by changing it to batchnorm2d(64), I noticed that the model quickly overfits. Is there a specific reason for this setting?
